mino/minoch/example_test.go

```diff
@@ -11,30 +11,13 @@ import (
 func ExampleRPC_Call() {
 	manager := NewManager()
 
-	minoA := MustCreate(manager, "A")
```

Comment:
yeah, I see now why you wanted to remove the return error, this example got a nice diet 😄